.storybook/3.COLORS.stories.mdx

```diff
@@ -74,7 +74,8 @@ var(--color-border-muted)
 
 /** Overlays */
 var(--color-overlay-default)
-var(--color-overlay-inverse) [DEPRECATED]
+var(--color-overlay-alternative)
+var(--color-overlay-inverse)
```

Comment:
Un-deprecating overlay/inverse as there are [use cases](https://consensys.slack.com/archives/C02VD8QG0LT/p1717083547070679) and adding overlay-alternative. See stylesheet in this branch

<img width="1512" alt="Screenshot 2024-06-09 at 8 08 00 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c4183dce-6923-4e3c-bde3-3cd6d061f7ee">